app/konnect/analytics/custom-reports/use-cases.md

```diff
@@ -0,0 +1,149 @@
+
+* What does average traffic over time look like for this service?
+* Are there any anomalies, such as spikes or drops in traffic?
+* If there are anomalies, do they repeat? Are there any patterns in the data?
+* If there's an issue with API traffic, has it been resolved or is it ongoing?
+
+#### Request per minute for the Account API over the last 30 days
+
+You can dive deeper into the Account API and build a report that displays the requests per minute for the Account API over the last 30 days. With this, you can develop a complete picture of the Accounts API, both on a per minute basis, and on a daily basis.
+
+* **Name**: Account API - RPM (last 30 days)
+
+* **Chart type**: Line chart
+* **Date/Time**: Last 30 days
+* **Metric**: Request count
+* **Group by**: Service
+
+### Daily API usage by application over the last 30 days
+
+At this point you have two custom reports: 
+
+* Total number of requests across APIs. 
+* Daily API Usage for an API over the last 30 days.
+
+These two custom reports can work together to provide you with a lot of information about your organizations product. These alone can help drive a lot internal change to your API. Let's dive deeper, if you want to learn about the types of third-party applications that your users use to connect to your API, set the following options in the UI: 
+
+* **Name**: API Usage by Application (last 30 days)
+* **Chart type**: Vertical bar chart
+* **Date/Time**: Last 30 days
+* **Metric**: Request count
+* **Group by**: Service
+* **Then by**: Application
+
+[screenshot of finished report]
+
+This report can be used to highlight which applications users prefer, where to distribute resources, and combined with the other reports, you can now communicate usage data about your API to your stakeholders.
+
+### Latency for an API over the last 30 days
+
+At this point, you have multiple reports that display how your company's APIs are performing generally.
+
+Your company determines that it is critical that payments are processed quickly using the Payments API, so you decide to generate a report that will tell you how performant the Payment API is over the last 30 days. To build the report, set the following options in the UI:
+
+* **Name**: Payment API - Latency (last 30 days)
+* **Date/Time**: Last 30 days
+* **Chart type**: Line chart
+* **Select a metric**: Response latency (p99)
+* **Group by**: Service
+* **Choose granularity**: Daily
+* **Entity Selection**: Payment 
+
+Add a filter for the Payment API. Click on **Add Filter**, then set the following options:
+ 
+* **Filter by**: Service
+* **Operator**: In
+* **Value**: Payment
+
+[screenshot of finished report]
+
+This report can provide your stakeholders with the answers to questions like:
+
+* How performant has my API been over the last 30 days?
+* Is my Payment API processing payments quickly enough?
+* Are there any performance outliers?
+
+## Conclusion 
```

Comment:
I wanted to relate this back to the KPIs we listed earlier, but wasn't sure if this was the best way or if the conclusion should contain something else?